### DISCUSSION TITLE
Lazer Tournament Client as a stand-alone client?

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#19042] [Ideas] [answered]
I think the Lazer Tournament Client being standalone would make it easy for people to add it to obs (because the lazer tournament Client would have the Client windows built in). But the thing about having the Client windows built in is that the osu!tourney (osu!stable tourney Client) the team size in the tournament.cfg wouldn’t really be necessary (nor would the entire stable tourney Client). But the fix for that would to add a slider to the lazer Client control panel that says, “Team Size,” and under it a slider for 1v1, 2v2, 3v3, and 4v4 tournament styles.

Hopefully people can add on to this. Thank you

**Comment by smoogipoo** [comment, marked as answer]
I'm not sure what this is requesting. Do you mean to display the gameplay in the lazer client? If so, it's planned: https://github.com/ppy/osu/issues/18896

**Reply by ghost** [reply]
Yes. That is what I mean. Whether it’s gameplay from osu!stable or osu!lazer.

**Reply by smoogipoo** [reply]
Gameplay from osu!lazer definitely. osu!stable likely not due to architectural differences.

**Reply by ghost** [reply]
That makes sense. Due to lazer being a COMPLETE rewrite of the code, the stable clients would break because the code is completely different. At the moment, the lazer client is basically a skin to the osu!tourney. To where people can add cool gfx to their tournaments.
